lib/middleware/get-user.js

```diff
@@ -112,48 +114,21 @@ module.exports = function (req, res, next) {
         });
       }
 
-      req.authenticationResult = authenticationResult;
-
-      // Backcompat: Provide the expanded JWT as req.accessToken,
-      // which has the claims under the 'body' key.
-      Object.defineProperty(req, 'accessToken', {
-        get: util.deprecate(function () {
-          var jwt = xtend({}, authenticationResult.expandedJwt);
-          jwt.body = xtend({}, jwt.claims);
+      req.authenticationResult = {
+        expandedJwt: jwt,
+        jwt: resolvedAccessToken
+      };
 
-          return jwt;
-        }, 'req.accessToken is deprecated, use req.authenticationResult instead.')
-      });
-
-      authenticationResult.getAccount(function (err, account) {
+      getAppUserById(config.okta.org, config.okta.appId, config.okta.apiToken, jwt.body.uid, (err, user) => {
         if (err) {
-          logger.info('Failed to get account ' + authenticationResult.account.href);
+          logger.info(err);
           return next();
         }
-
-        if (account.status !== 'ENABLED') {
```

Comment:
Right.  This is a strange check, I'm not sure why it was here in the first place.  In this context the authentication attempt has already happened, which would have failed if the user were not enabled.